nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -55,7 +55,7 @@ WaypointFollower::on_configure(const rclcpp_lifecycle::State & /*state*/)
   stop_on_failure_ = get_parameter("stop_on_failure").as_bool();
   loop_rate_ = get_parameter("loop_rate").as_int();
   waypoint_task_executor_id_ = get_parameter("waypoint_task_executor_plugin").as_string();
-
+  // related to waypoint following
```

Comment:
I'm not sure this comment is needed, its actually before the node creation so its misplaced if nothing else (and missing a GPS analog).

Also, if you want to move & keep, please add a new line before it for readability.